pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -269,6 +269,40 @@ def test_to_datetime_utc_is_true(self):
         result = pd.to_datetime(date_range, utc=True)
         expected = pd.DatetimeIndex(data=date_range)
         tm.assert_index_equal(result, expected)
+    
+    @pytest.mark.parametrize("init_constructor, end_constructor, test_method",
+        [(Index, DatetimeIndex, tm.assert_index_equal),
+         (Series, Series, tm.assert_series_equal)])
+    def test_to_datetime_utc_true_with_series(self, 
```

Comment:
you can remove the 'with series' in the name I think, as it is now more general test of utc ?